src/js/actions/queries.actions.js

```diff
@@ -0,0 +1,15 @@
+import {QUERIES} from "../constants/action.constants";
+
+export const addQuery = query => {
+    return {
+        type: QUERIES.ADD,
+        payload: query
+    }
+};
+
+export const deleteQuery = time => {
+    return {
+        type: QUERIES.DELETE,
+        payload: time
+    }
+};
```

Comment:
End of file should have newline to easier merging 

<img width="296" alt="screen shot 2017-10-30 at 11 45 44" src="https://user-images.githubusercontent.com/2149114/32164341-e3dfd3aa-bd67-11e7-88d7-cc95132953e7.png">

Reply:
I will fix it. Thanks for advice.